raiden/tests/unit/transfer/mediated_transfer/test_targetstate.py

```diff
@@ -332,6 +333,7 @@ def test_handle_block():
         from_channel,
         pseudo_random_generator,
         new_block.block_number,
+        UNIT_CHAIN_ID,
```

Comment:
why are you passing the chain_id as an argument for the target state machine?

it should be available through the signed balance proof

Reply:
yeah that should not be there. Not even used in the target state machine as I use it from the signed BP as you said. Justa remnant from before.